index.html

```diff
@@ -807,7 +810,8 @@
       property to <code>false</code> and set up a watch before the push.
     </p>
     <pre class="highlight">
-navigator.nfc.watch((message) => {
+const reader = new NFCReader();
+reader.watch(message => {
```

Comment:
https://github.com/w3c/web-nfc/pull/179/files#r239426031

Reply:
OK will fix the samples!